pkg/installer/deployresources_resources.go

```diff
@@ -106,7 +106,7 @@ func (m *manager) computeBootstrapVM(installConfig *installconfig.InstallConfig)
 	vm := &mgmtcompute.VirtualMachine{
 		VirtualMachineProperties: &mgmtcompute.VirtualMachineProperties{
 			HardwareProfile: &mgmtcompute.HardwareProfile{
-				VMSize: mgmtcompute.VirtualMachineSizeTypesStandardD4sV3,
+				VMSize: mgmtcompute.VirtualMachineSizeTypesStandardD8sV3,
```

Comment:
I think instead of a hard-coded value, we'll want to use whatever the Master size is, which is variable based on customer selection. Can we update this to use whatever was passed in as the master size?

Reply:
In addition, this change shouldn't go in the "old" installer code, let's put it into the 4.11 branch on the ARO-Installer repo: https://github.com/openshift/ARO-Installer/blob/main/pkg/installer/deployresources_resources.go